src/main/java/bio/terra/common/iam/TokenAuthenticatedRequestFactory.java

```diff
@@ -0,0 +1,28 @@
+package bio.terra.common.iam;
+
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.stereotype.Component;
+
+/**
+ * Implementation of {@link AuthenticatedUserRequestFactory} when HTTP requests contain a bearer
+ * token in the Authorization header.
+ *
+ * <p>This factory is all that should be required to propagate the caller's identity in most cases.
+ * In cases where Sam is not otherwise called to check the caller's access or the caller's email or
+ * id are required, seel{@link SamAuthenticatedUserRequestFactory}.
+ */
+@Component
+public class TokenAuthenticatedRequestFactory {
+
+  static final String AUTHORIZATION = "Authorization";
```

Comment:
It looks like this is only visible for testing purposes
```suggestion
  @VisisbleForTesting
  static final String AUTHORIZATION = "Authorization";
```